src/main/groovy/org/mockito/release/internal/gradle/AutoVersioningPlugin.java

```diff
@@ -28,67 +28,23 @@
  * Adds following tasks:
  *
  * <ul>
- *     <li>gitAddBumpVersion</li>
- *     <li>gitCommitBumpVersion</li>
- *     <li>gitPushBumpVersion</li>
  *     <li>bumpVersionAndPush</li>
  * </ul>
  */
 public class AutoVersioningPlugin implements Plugin<Project> {
 
     static final String BUMP_VERSION_AND_PUSH_TASK = "bumpVersionAndPush";
-    static final String ADD_BUMP_VERSION_TASK = "gitAddBumpVersion";
-    static final String PUSH_BUMP_VERSION_TASK = "gitPushBumpVersion";
-    static final String VERSION_BUMP_COMMIT_TASK = "gitCommitBumpVersion";
 
     public void apply(final Project project) {
```

Comment:
It's great when we have lots of plugins and top-level plugins mostly apply some other fine-grained plugins. It will make the codebase easier to work with and make the tool really flexible for advanced users. This plugin is a good example - mostly it applies other plugins. I would love the "continuous-delivery" plugin to be like that - only applies other plugins, zero other code.